specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2024-05-01-preview/azureopenai.yaml

```diff
@@ -0,0 +1,1890 @@
+          connectionString:
+            type: string
+            description: Connection string
+    x-ms-discriminator-value: connectionString
+  EndpointMIConnection:
+    description: Endpoint Managed Identity connection.
+    required:
+      - endpoint
+    allOf:
+      - $ref: '#/definitions/BaseConnection'
+      - properties:
+          endpoint:
+            type: string
+            description: Endpoint
+    x-ms-discriminator-value: endpointMI
+  WorkspaceConnection:
+    description: AML Workspace connection.
+    required:
+      - connectionId
+    allOf:
+      - $ref: '#/definitions/BaseConnection'
+      - properties:
+          connectionId:
+            type: string
+            description: ConnectionId
+    x-ms-discriminator-value: workspace
+  DeploymentConnection:
+    description: Relative deployment connection.
+    allOf:
+      - $ref: '#/definitions/BaseConnection'
+    x-ms-discriminator-value: relativeDeployment
+  CompletionAction:
+    type: string
+    description: The completion action.
+    enum:
+      - cleanUpAssets
+      - keepAllAssets
+    x-ms-enum:
+      name: IngestionJobCompletionAction
+      modelAsString: true
+      values:
+        - name: keepAllAssets
+          value: keepAllAssets
+          description: Will not clean up any of the intermediate assets created during the ingestion process.
+        - name: cleanUpAssets
+          value: cleanUpAssets
+          description: Will clean up intermediate assets created during the ingestion process.
+  IngestionJobList:
+    title: IngestionJobList
+    description: Represents a list of ingestion jobs.
+    type: object
+    properties:
+      nextLink:
+        description: The link to the next page.
+        type: string
+      value:
+        description: The list of items.
+        type: array
+        items:
+          $ref: '#/definitions/IngestionJob'
+  IngestionJobRunList:
+    title: IngestionJobRunList
+    description: Represents a list of ingestion job runs.
+    type: object
+    properties:
+      nextLink:
+        description: The link to the next page.
+        type: string
+      value:
+        description: The list of items.
+        type: array
+        items:
+          $ref: '#/definitions/IngestionJobRun'
+  IngestionJobRun:
+    title: IngestionJobRun
+    description: The details of a job run
+    type: object
+    properties:
+      jobId:
+          type: string
+      runId:
+          type: string
+      status:
+        $ref: '#/definitions/OperationState'
+      error:
+        $ref: '#/definitions/IngestionError'
+      warnings:
+        type: array
+        items:
+          type: string
+      progress:
+        $ref: '#/definitions/IngestionJobProgress'
+  IngestionError:
+    title: IngestionError
+    description: The details of the ingestion error
+    type: object
+    properties:
+      message:
+        type: string
+      innerErrors:
+        type: array
+        items:
+        type: string
```

Comment:
Sorry for commenting on a closed PR, this entity is flagged as incorrectly defined. It has 2 `type` keys.

Reply:
@YunsongB , just @'ing you for visibility :)